app/main.py

```diff
@@ -14,12 +14,12 @@ def __init__(self, distance_from_city_center: float,
         self.average_rating = average_rating
         self.count_of_ratings = count_of_ratings
 
-    def calculate_washing_price(self, car: Car) -> float:
+    def calculate_washing_price(self, car: list[Car]) -> float:
         price = (car.comfort_class * (self.clean_power - car.clean_mark)
                  * self.average_rating / self.distance_from_city_center)
         return round(price, 1)
 
-    def wash_single_car(self, car: Car) -> str:
+    def wash_single_car(self, car: list[Car]) -> str:
```

Comment:
list[Car] only when function gets many cars - list of cars, please return previous data. here it must be car: Car